common/src/main/java/com/wynntils/screens/maps/GuildMapScreen.java

```diff
@@ -55,7 +64,9 @@ public final class GuildMapScreen extends AbstractMapScreen {
     private BasicTexturedButton territoryDefenseFilterButton;
     private BasicTexturedButton hybridModeButton;
 
-    private GuildMapScreen() {}
+    private GuildMapScreen() {
+        this.firstInit = true;
```

Comment:
Sorry for the nit, but could you move setting this to the field declaration? That would make it like other screens. Thanks

Reply:
I think I've seen another screen do it this way but maybe I'm wrong, can't find it now to fix here